helpers/auth/admin_roles.py

```diff
@@ -40,12 +48,6 @@ def update_delete_floor(self, floor_id):
         if admin_details.location != floor_location.name:
             raise GraphQLError("You are not authorized to make changes in " + floor_location.name)  # noqa: E501
 
-    def create_office(self, location_id):
```

Comment:
I removed this because it was a duplicate function and used one with a better descriptive name